inception/inception-experimental-editor/src/main/java/ts/editors/wordalignment/AnnotationExperienceAPIWordAlignmentEditor.ts

```diff
@@ -0,0 +1,144 @@
+                        document.getElementById("container_german").children[j + 2].children[0].id.split("_")[2],
+                        document.getElementById("container_german").children[j + 2].children[0].innerText]
+                        );
+                    this.annotationExperienceAPI.requestCreateSpanFromServer(
+                        that.annotationExperienceAPI.clientName,
+                        that.annotationExperienceAPI.clientName,
+                        that.annotationExperienceAPI.projectID,
+                        this.annotationExperienceAPI.documentID,
+                        Number(document.getElementById("container_english").children[i + 2].children[1].id.split("_")[2]),
+                        Number(document.getElementById("container_english").children[i + 2].children[1].id.split("_")[3]),
+                        "webanno.custom.Word_Alignment_Span")
+
+                    this.annotationExperienceAPI.requestCreateSpanFromServer(
+                        that.annotationExperienceAPI.clientName,
+                        that.annotationExperienceAPI.clientName,
+                        that.annotationExperienceAPI.projectID,
+                        this.annotationExperienceAPI.documentID,
+                        Number(document.getElementById("container_german").children[i + 2].children[0].id.split("_")[2]),
+                        Number(document.getElementById("container_german").children[i + 2].children[0].id.split("_")[3]),
+                        "webanno.custom.Word_Alignment_Span")
+
+                    document.getElementById("container_german").children[j + 2].children[1].setAttribute("disabled", "true");
+                }
+            }
+        }
+
+        setTimeout(function () {
+            let governor, dependent;
+            for (let i = 0; i < pairs.length; i++) {
+                for (let j = 0; j < that.annotationExperienceAPI.spans.length; j++) {
+                    if (pairs[i][0] == that.annotationExperienceAPI.spans[j].begin &&
+                        pairs[i][1] == that.annotationExperienceAPI.spans[j].coveredText) {
+                        governor = that.annotationExperienceAPI.spans[j];
+                    }
+                    if (pairs[i][2] == that.annotationExperienceAPI.spans[j].begin &&
+                        pairs[i][3] == that.annotationExperienceAPI.spans[j].coveredText) {
+                        dependent = that.annotationExperienceAPI.spans[j];
+                    }
+                }
+                that.annotationExperienceAPI.requestCreateRelationFromServer(
+                    that.annotationExperienceAPI.clientName,
+                    that.annotationExperienceAPI.clientName,
+                    that.annotationExperienceAPI.projectID,
+                    that.annotationExperienceAPI.documentID,
+                    governor.id,
+                    dependent.id,
+                    "webanno.custom.Word_Alignment_Relation")
```

Comment:
Hm yeah, for the span its pretty simple by using the selected layer in the annotation sidebar. However, shouldn't INCEpTION maybe have a separate span/relation layer for word_alignments? Because this layer may only be required by that kind of editors and the only other standard relation layer is "Dependency" based on POS which would not make sense here.

Reply:
We have different types of layers including span and relation. Currently, there can be exactly one relation layer per span layer, meaning that if you choose a relation layer, you implicitly also choose the span layer. The question then is "which layer configurations is an editor compatible with?". E.g. your bi-partite graph annotation editor is probably compatible with and relation layer that does not allow crossing sentence boundaries and which attaches to a span layer that has token-level granularity and also does not allow crossing sentence boundaries. If INCEpTION comes with a compatible pre-defined layer is another question - maybe yes, maybe no. But a user should be able to define a custom layer that is compatible with whatever you define and then use that.